trikot-kword/sample/common/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
     id("com.android.library")
     kotlin("multiplatform")
     kotlin("native.cocoapods")
-    id("mirego.kword").version("5.5.0-dev2726")
+    id("mirego.kword").version("5.5.0-SNAPSHOT")
```

Comment:
This wont work unless you ran publishToMavenLocal before. So it would work with CI with what you changed but not our machines.
Lets revert that.